R/workhorse.R

```diff
@@ -75,22 +75,24 @@ make_request <- function(x, ...){
 }
 
 process_request <- function(res,
-                            content = TRUE,
-                            expected = "application/json; charset=UTF-8") {
-  httr::stop_for_status(res)
-  if (content == TRUE) {
-    actual <- res$headers$`content-type`
-    if (actual != expected) {
-      spf(
-        paste0(
-          "Expected content-type:\n%s",
-          "\n",
-          "Actual content-type:\n%s"
-        ),
-        expected,
-        actual
-      )
-    }
-    jsonlite::fromJSON(httr::content(res, "text"), simplifyVector = FALSE)
+                            expected = "application/json; charset=UTF-8",
+                            internet = TRUE) {
```

Comment:
hmm yes that makes sense. In order to get an object that `httr::stop_for_status()` and `httr::content()` won't error on, I was thinking we needed the internet (unless we save some object that httr recognizes as a response?)